ssh/tailssh/incubator.go

```diff
@@ -124,10 +124,9 @@ func (ss *sshSession) newIncubatorCommand() (cmd *exec.Cmd) {
 	} else {
 		if isShell {
 			incubatorArgs = append(incubatorArgs, "--shell")
-			// Currently (2022-05-09) `login` is only used for shells
-			if lp, err := exec.LookPath("login"); err == nil {
-				incubatorArgs = append(incubatorArgs, "--login-cmd="+lp)
-			}
+		}
+		if lp, err := exec.LookPath("login"); err == nil {
+			incubatorArgs = append(incubatorArgs, "--login-cmd="+lp)
```

Comment:
commit message says "on macOS" but this doesn't look like macOS-specific code? you changing Linux behavior here?

Reply:
it adds a lookup but doesn't change behavior, will add an additional check

Reply:
Done